compiler-plugin/testData/api/overridee.kt

```diff
@@ -295,3 +297,8 @@ class JavaAccessorImpl implements KtInterfaceWithProperty {
     public void setY(int value) {
     }
 }
+
+// FILE: JavaAnno.java
+@interface JavaAnno {
```

Comment:
It might be good to test repeating annotations as well, but can be done in a follow-up PR.

Reply:
added, tbh I still don't know what differs by being repeating.

Reply:
I expected to see array return type. :)

Reply:
Ugh... That should be in another test that is checking for jvm name. This test only checks for override.